framework/test-vectors/test-vector-enumeration.md

```diff
@@ -0,0 +1,176 @@
+[//]: # "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved."
+[//]: # "SPDX-License-Identifier: CC-BY-SA-4.0"
+
+# Test Vector Enumeration 
+
+## Overview
+
+Test vectors are a suite of known value tests.
+Successfully running test vectors asserts that
+given some set of input configurations to encrypt and/or decrypt,
+when these configurations are encrypted and/or decrypted,
+then the expected result is achieved on encrypt and/or decrypt.
+
+This document outlines a framework
+to enumerate a set of pairings
+of input configurations to encryption and/or decryption
+and expected results of encryption and/or decryption.
+The framework constructs this set of pairings
+to be representative of all possible
+input configurations and expected results.
```

Comment:
Lets say a little more here about the diff between "all possible" and "representative"